jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -48,107 +49,187 @@ class LEJExtensionContext {
 const extensionContext = new LEJExtensionContext()
 registerExtensions(extensionContext)
 
-class HitherJobManager {
-  _activeJobs: {[key: string]: any} = {}
-  _jobFinishedCallbacks: {[key: string]: (result: any, runtime_info: any) => void} = {}
-  _jobErrorCallbacks: {[key: string]: (error_message: string, runtime_info: any) => void} = {}
-  _iterating: boolean = false
-  constructor(private model: DOMWidgetModel) {
-    model.on('msg:custom', (msg: any) => {
-      if (msg.type === 'hitherJobFinished') {
-        const clientJobId = msg.client_job_id
-        if (this._activeJobs[clientJobId]) {
-          const cb = this._jobFinishedCallbacks[clientJobId]
-          if (cb) {
-            cb(msg.result, msg.runtime_info)
-          }
-          delete this._activeJobs[clientJobId]
-        }
+// class HitherJobManager {
```

Comment:
The shared hither job management code between jupyter widget and the web gui widget has been expanded, so there is a lot less code that needs to be separately maintained. As part of this consolidation, this HitherJobManager class is being removed.